codespell_lib/data/dictionary.txt

```diff
@@ -25579,6 +25591,7 @@ wikpedia->wikipedia
 wil->will, well,
 wilcard->wildcard
 wilcards->wildcards
+wile->while
```

Comment:
`wile` is a somewhat rare but not totally obscure real word

Reply:
disabled then ? I see it frequently in https://github.com/search?q=wile&type=Wikis